queries.sql

```diff
@@ -33,3 +33,63 @@ WHERE NAME <> 'Gabumon';
 SELECT *
 FROM animals
 WHERE weight_kg BETWEEN 10.4 AND 17.3;  
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+COMMIT;
+SELECT * FROM animals;
+
+UPDATE animals
+SET species = 'pokemon';
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE
+FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+-- this will show that the table exists
+\d animals
+
+BEGIN;
+DELETE
+FROM animals
+WHERE date_of_birth > 'January 1, 2022';
+SAVEPOINT younger_deleted;
+
+UPDATE animals
+SET weight_kg = weight_kg * (-1);
+ROLLBACK TO younger_deleted;
+UPDATE animals
+SET weight_kg = weight_kg * (-1)
+WHERE weight_kg < 0;
+COMMIT;
+
+SELECT count(*) FROM animals;
+SELECT count(*) FROM animals
+WHERE escape_attempts = 0;
+
+SELECT AVG(weight_kg) AS average_weight FROM animals;
+SELECT * FROM animals
+ORDER BY escape_attempts DESC
+LIMIT 1;
```

Comment:
- [ ] This query suppose to answer `Who escapes the most, neutered or not neutered animals?`, not who animal escapes the most? Please fix that.
[Hint] It's supposed to be like that:
```sql
SELECT neutered, MAX(escape_attempts) FROM animals
GROUP BY neutered;
```